### DISCUSSION TITLE
Nginx image refuses to start up for only one of our users

### DISCUSSION THREAD

**Discussion by jbxonline** [serversideup/docker-php#414] [Q&A] [answered]
Hey there!  
We are currently using `serversideup/php:7.4-fpm-nginx` as our base image and it runs perfectly for all but one of our devs.  
He is using the same type of M1 Mac as me, same Docker version, same checked out files.  
However, when he starts up, he gets this in his Docker Inspect panel:
```
"Health": {
    "Status": "unhealthy",
    "FailingStreak": 36,
    "Log": [
        {
            "Start": "2024-08-19T16:08:31.667416466Z",
            "End": "2024-08-19T16:08:31.777787716Z",
            "ExitCode": 1,
            "Output": "curl: (7) Failed to connect to localhost port 8080: Connection refused\n"
        },
```
I cannot understand how the call to port 8080 could fail from within the docker image.  
If anyone has any clues or ideas to try, please do let me know as right now I'm at a bit of a loss as to what could cause this.  
The image at this point is essentially running, but nginx never starts as the boot up process never completes. I can, however, log into the container to check things.

Thanks in advance!

Jon

**Comment by jaydrogers** [comment]
That's super strange! I just ran this on my M1 and it came up okay:

```bash
docker run --rm serversideup/php:7.4-fpm-nginx
```

Things I would check:
- Ensure latest version of Docker is running
- Do they have any special security settings?
- Ensure operating system updates are installed
- Try deleting and re-installing Docker desktop

Hope this helps! I wish I ran into this issue before to be more valuable 😃

**Reply by jbxonline** [reply, marked as answer]
I actually got to the bottom of this just before you posted.  
Nothing to do with your images you'll be pleased to know.  
The dev who had the issue had an Xdebug config setting `XDEBUG_START_WITH_REQUEST` set to `yes` instead of trigger. And as his IDE was set to listen, it was starting an Xdebug session during the container init sequence. Essentially causing it to stop and wait. Managed to replicate it, changed the setting and suddenly everything sprung into life :)
